fuel-vm/src/gas/default_gas_costs.rs

```diff
@@ -0,0 +1,122 @@
+use super::*;
+
+pub const GIT: &str = "1a660f59b892e7816da01c288c86696c9d7d7a36";
```

Comment:
A comment would be nice to document exactly what this is referencing

Reply:
This is generated code so I would need to add this in the fuel-core PR

Reply:
Maybe the codegen could inject some comments?

E.g. 

```
format!("//Generated by fuel-core: {}:{}", file!(), line!())
```

Reply:
Ok made an issue for this